cms/envs/common.py

```diff
@@ -342,6 +342,47 @@
     # .. toggle_status: supported
     # .. toggle_warnings: None
     'ENABLE_CHANGE_USER_PASSWORD_ADMIN': False,
+
+    ### ORA Feature Flags ###
+
+    # .. toggle_name: ENABLE_ORA_TEAM_SUBMISSIONS
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to enable team-based ORA submissions.
+    # .. toggle_category: ora
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2020-03-03
+    # .. toggle_expiration_date: None
+    # .. toggle_tickets: https://openedx.atlassian.net/browse/EDUCATOR-4951
+    # .. toggle_status: supported
+    # .. toggle_warnings: None
+    'ENABLE_ORA_TEAM_SUBMISSIONS': False,
+
+    # .. toggle_name: ENABLE_ORA_ALL_FILE_URLS
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: A "work-around" feature toggle meant to help in cases where some file uploads are not discoverable.
+    # .. toggle_category: ora
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2020-03-03
+    # .. toggle_expiration_date: None
+    # .. toggle_tickets: https://openedx.atlassian.net/browse/EDUCATOR-4951
+    # .. toggle_status: supported
+    # .. toggle_warnings: None
+    'ENABLE_ORA_ALL_FILE_URLS': False,
+
+    # .. toggle_name: ENABLE_ORA_USER_STATE_UPLOAD_DATA
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: A "work-around" feature toggle meant to pull file upload data our of user state, rather than Submission records.
+    # .. toggle_category: ora
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2020-03-03
+    # .. toggle_expiration_date: None
+    # .. toggle_tickets: https://openedx.atlassian.net/browse/EDUCATOR-4951
+    # .. toggle_status: supported
+    # .. toggle_warnings: None
+    'ENABLE_ORA_USER_STATE_UPLOAD_DATA': False,
```

Comment:
I don't fully grasp what these are for.

Are they temporary toggles that will be removed when ORA is done being actively developed? If so, a [different value for toggle_use_cases](https://github.com/edx/edx-toggles/blob/master/feature_toggle_annotations.yaml#L19) might be warranted.

Or do they exist as to not break Open edX ORA users? If so, can you leave an explanation that would help an Open edX operator decide whether to turn them on or not?

Reply:
The `open_edx` use case is copypasta, I wasn't sure how general purpose it was meant to be.  I'm not sure that any of the use case choices you linked to really fit the actual use case - these are toggles that enable a workaround for a specific failure mode.  I'm also not sure to what degree we make use of these annotations, so maybe I'll omit`toggle_use_cases` for these two.  I'll also update the description to be more clear about why these exist.